### PR TITLE
DHFPROD-8381: Card View not toggling from Graph View

### PR DESCRIPTION
**Pull request by iveyMarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [N/A] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [N/A] Reviewed Tests
- [N/A] Added Release Notes

### CODE REVIEW

**Review by briantang — DISMISSED** [review]
Looks fixed.

**Review by Sanjeevani19 — DISMISSED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@iveyMarklogic Can we add an e2e for this scenario ?

**Review by xnikhil08 — DISMISSED** [review]

**Comment by iveyMarklogic** [comment]
@Sanjeevani19 I added e2es to cover those scenarios

**Review by aniamy — APPROVED** [review]

**Comment by aniamy** [comment]
Toggling now keeps a memory of the previous choice in Entities for all options. Looks good.

**Review by briantang — DISMISSED** [review]
Nice @iveyMarklogic looks good!

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/exploreViewSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    //Saving Local Storage to preserve session
+    cy.saveLocalStorage();
+  });
+  beforeEach(() => {
+    //Restoring Local Storage to Preserve Session
+    Cypress.Cookies.preserveOnce("HubCentralSession");
+    cy.restoreLocalStorage();
+  });
+  it(`Validate that the 'graph' view is shown and stored in the user preference`, () => {
+    cy.log(`**Go to Explore section?**`);
```
Comment (csilva-ml) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explore/exploreViewSelector.tsx:27]: Do you mind removing the "?" ?

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/exploreViewSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    toolbar.getExploreToolbarIcon().click();
+
+    cy.log(`**Selecting 'Graph' view**`);
+    browsePage.viewSelector("graph").should("exist").click();
+    browsePage.getGraphVisExploreContainer().should("exist");
+
+    cy.log(`**Go to Monitor section**`);
+    toolbar.getMonitorToolbarIcon().click();
```
Comment (csilva-ml) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explore/exploreViewSelector.tsx:35]: You can add a simple validation here simply to confirm that it actually went to monitor section. For example a title

**Review by csilva-ml — CHANGES_REQUESTED** [review]
Great work on the e2e, just some minor requests on the validations, thank you!

**Review by aniamy — APPROVED** [review]

**Review by xnikhil08 — APPROVED** [review]

**Pull request merged**